discovery-provider/default_config.ini

```diff
@@ -49,5 +49,5 @@ allow_all = false
 registry = 0x2999e02829DC711B9254187962ba44d1fFcf5481
 
 [delegate]
-owner_wallet =
-private_key =
+owner_wallet = 0x1D9c77BcfBfa66D37390BF2335f0140979a6122B
+private_key = 0x3873ed01bfb13621f9301487cc61326580614a5b99f3c33cf39c6f9da3a19cad
```

Comment:
why is this necessary?

Reply:
i'm actually super unhappy with us having committed private keys even if they are dummy it's just such a bad practice

Reply:
would be better to remove them from .env as well

Reply:
yeah i agree, but right now the prod-web-server script doesn't work unless we have some values here (and i just copied these from the dummy ones in .env). if ur super against i can just close the PR nbd. we should clean up the other spots then too tho

Reply:
jw what environment doesn't consume the defaults in .env?

Reply:
Sometimes i like running a discprov against staging dbs -- using the prod-web-server.sh script (which just spins the stuff up in docker). Maybe all this should just be in the tooling repo anyway

Reply:
i usually run the compose files with a specific service name which avoids this - like `docker-compose -f docker-compose.base.yml -f docker-compose.dev.yml up -d web-server`